doc/book/examples/sources/basics/publisher.move

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// A simple package that defines an OTW and claims a `Publisher`
+/// object for the sender.
+module examples::owner {
+    use sui::tx_context::TxContext;
+    use sui::package;
+
+    /// OTW is a struct with only `drop` and is named
+    /// after the module - but uppercased. See "One Time
+    /// Witness" page for more details.
+    struct OWNER has drop {}
+
+    /// Some other type to use in a dummy check
+    struct ThisType {}
+
+    /// After the module is published, the sender will receive
+    /// a `Publisher` object. Which can be used to set Display
+    /// or manage the transfer policies in the `Kiosk` system.
+    fun init(otw: OWNER, ctx: &mut TxContext) {
+        package::claim_and_keep(otw, ctx)
+    }
+}
+
+/// A module that utilizes the `Publisher` object to give a token
+/// of appreciation and a `TypeOwnerCap` for the owned type.
+module examples::type_owner {
```

Comment:
I think this example is a bit hard to follow without seeing a usage of the `TypeOwnerCap`.

In general, I think it's most useful for these examples to show how to write "product-y" code (and then show how to use it). "Infra -y" code like this is also great, but is targeting a somewhat smaller/different audience I think.

Reply:
I think `Publisher` is the Infra-y (reusable) part here -- @damirka, do you have any interesting product-specific examples (outside of Display) for wanting to know that a caller created the type?

Reply:
Answered below. There's no simple application for a Publisher as it is a creator-specific feature to show that they do own a type and that they have control over them.